src/averageAges.js

```diff
@@ -37,7 +47,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womenAgeSum = people
+    .filter(person =>
```

Comment:
as hint: you can use destructuring
```suggestion
    .filter(({ name, sex }) =>
```